src/PythonClient/src/quixstreams/models/timeseriesdataraw.py

```diff
@@ -80,11 +80,11 @@ def __str__(self):
 
         return text
 
-    def to_panda_dataframe(self) -> pd.DataFrame:
+    def to_dataframe(self) -> pd.DataFrame:
         """
-        Converts TimeseriesDataRaw to Panda DataFrame
+        Converts TimeseriesDataRaw to pandasDataFrame
 
-        :return: Converted Panda DataFrame
+        :return: Converted pandasDataFrame
```

Comment:
same pandasDataFrame VS pandas Dataframe debate

Reply:
thought I found all these, but apparently not. Just a typo, fixing